### ISSUE TITLE
Deprecation of request.connection

### ISSUE THREAD

**Issue by birkhofflee** [supercharge/request-ip#4]
In https://nodejs.org/api/http.html#http_request_connection, now the conventional object to get the IP address is `request.socket`. Usage of this module with Fastify also introduces a FastifyDeprecation warning:

```
(node:95815) [FSTDEP005] FastifyDeprecation: You are accessing the deprecated "request.connection" property. Use "request.socket" instead.
```

**Comment by marcuspoehls** [comment]
@BirkhoffLee Hey, thank you for the merge request resolving this deprecation warning 👍  I’ll publish a new release today

**Comment by marcuspoehls** [comment]
I’ve published version `1.2.0` to npm. Thank you again 🤝 😃
